plugins/pulp_rpm/plugins/migrations/0027_remove_checksum_type_field.py

```diff
@@ -0,0 +1,21 @@
+"""
+This migration removes the `checksum_type` field of the units_rpm collection.
+"""
+from pulp.server.db import connection
+
+
+def migrate(*args, **kwargs):
+    """
+    Perform the migration as described in this module's docblock.
+
+    :param args:   unused
+    :type  args:   list
+    :param kwargs: unused
+    :type  kwargs: dict
+    """
+    db = connection.get_database()
+    collection = db['units_rpm']
+    collection.update(
+        {"checksum_type": {"$exists": True}},
```

Comment:
I assume there's not an index on this field. I know mongo will sometimes refuse to do things like sort on a field that's not indexed if there are too many matched results. Will it complain that way for just a normal query like this? Or can we expect this to succeed with an unlimited number of matching documents?